namespaces/live.cloud-platform.service.justice.gov.uk/court-probation-e2e/resources/variables.tf

```diff
@@ -0,0 +1,32 @@
+variable "vpc_name" {
+}
+
+variable "application" {
+  default = "HMPPS  Court Probation"
```

Comment:
Are the additional spaces a typo or a requirement?

Reply:
Looks like a typo. will remove.Ta